content/How to Test React Native Apps - Manuela Viola.md

```diff
@@ -0,0 +1,143 @@
+# How to Test React Native Apps
+
+## Introduction
+
+Have you ever had a bug introduced by solving another problem and didn't realize it until weeks later? This could have been easily avoided if you had unit tests for your App.
```

Comment:
is this article about unit tests only? it'd seem that further in the article you are talking about UI tests since you are testing navigation and UI components besides models. 
Also, in most cases and especially for front end apps, unit testing is not enough and you want to start looking into UI and integration testing.

Reply:
yeah, unit wasn't the best word there, thanks.